### DISCUSSION TITLE
Non-individual person records

### DISCUSSION THREAD

**Discussion by MarcinKrawczuk** [SChAth/ScAthiop#1606] [Q&A] [answered]
Is it acceptable to create person records for a group of people who always appear together? A good example are saints e. g. ቆጵርያኖስ፡  and  ኢዩስጣ for whom IMHO it would be superflous to create individual records for each of them. I thound this in the guidelines: https://betamasaheft.eu/Guidelines/?q=group&id=groups but that's a different case, I think.  This one is probably what I have in mind https://betamasaheft.eu/persons/PRS11288AnAzMi/main, but is it really necessary to create individual records?

@eu-genia @thea-m @DariaElagina

**Comment by eu-genia** [comment, marked as answer]
No, I believe groups is a different case, your case is like Cyriacus and Julitta - they always go together and still each has his/her own record.

**Comment by PietroLiuzzo** [comment]
Dear Marcin yes, it is necessary for consistency to stick with this definitions and tag each name, otherwise when referring to a person record we will never know if it actually is one, two, three persons. 
In the groups, as you have already found out, we use `<personGrp>` and a `<relation>` element to organize that information. You can use a relation between the two records if you want. On the other side, you do not have to tag every occurrence of a person name, you can, but do not have to.

**Comment by MarcinKrawczuk** [comment]
Thank you, that's what I wanted to know.
